apps/glusterfs/volume_entry_test.go

```diff
@@ -2901,6 +2901,62 @@ func TestVolumeCreateTooFewZones(t *testing.T) {
 		err = v.Create(app.db, app.executor)
 		tests.Assert(t, err == nil, "expected err == nil, got:", err)
 	})
+
+	t.Run("VolOpt ZoneChecking none replica 3", func(t *testing.T) {
+		ZoneChecking = ZONE_CHECKING_STRICT
+		// Can't fit a replica-3 volume into 2 zones with strict zone checking
+		req := &api.VolumeCreateRequest{}
+		req.Size = 10
+		req.Durability.Type = api.DurabilityReplicate
+		req.Durability.Replicate.Replica = 2
+		req.GlusterVolumeOptions = []string{"user.heketi.zone-checking none"}
+		v := NewVolumeEntryFromRequest(req)
+
+		err = v.Create(app.db, app.executor)
+		tests.Assert(t, err == nil, "expected err == nil, got:", err)
+	})
+
+	t.Run("VolOpt ZoneChecking invalid replica 3", func(t *testing.T) {
+		ZoneChecking = ZONE_CHECKING_NONE
+		// Can't fit a replica-3 volume into 2 zones with strict zone checking
```

Comment:
I realize this is an inappropriate comment that was carried over by copy'n'paste from the original PR #1518 ... will fix that PR.